src/content/blog/2024/special_interest_groups.mdx

```diff
@@ -0,0 +1,155 @@
+from disparate groups comparable and reproducible.
+
+To meet this need, we want to provide a space for community members working on
+similar research interests to gather and collaborate.
+We want to foster collaboration not just in the development of scientific
+research tooling, but also in the usage of that tooling across multiple fields.
+We hope that special interest groups will provide a place for this to happen.
+
+# How will this work?
+
+We like how our community is structured around pipelines.
+Our established methods for these groups appears to work well, so rather than
+reinventing the wheel we intend to adopt a similar pattern for special interest groups:
+
+1. Anyone can suggest a new group in the `#new-interest-groups` Slack channel
+    - Just like with `#new-pipelines`, there is a Slack workflow with a form to fill in,
+      with a handful of questions to collect all the required information.
+2. The core team (and steering group, as necessary) will discuss the proposal, suggest
+   any changes and either accept or reject the proposal.
+    - This process will be tracked in a GitHub project, called
+      [New interest groups](https://github.com/orgs/nf-core/projects/63)
+3. A new Slack channel will be created for the new interest group on the nf-core Slack
+4. A new web page will be created for the group on the nf-core website
+5. The lead for the new group will organise a nf-core/bytesize talk to introduce
+   the group to the community. A blog post will also be recommended.
+6. The new special interest group will be active and continue under their own steam.
+
+## Group web pages
+
+The new web page will just be a stub at first, but we hope that the interest group
+will develop this over time. These pages can evolve into mini sections with multiple
+sub-pages, including things like best-practice documentation, meeting notes and more.
+
+We would like to standardise a listing of the nf-core pipelines typically used by
+members of each interest group, so that the pages can be cross-linked.
+That way, anyone viewing a web page for an nf-core pipeline will be able to see
+at a glance what it can be used for and hopefully link to best practices for
+that pipeline with that type of data.
+
+## Closing after inactivity
+
+Just like pipelines, special interest groups will never be _deleted_.
+As part of the annual "spring cleaning" that the nf-core maintainers team does,
+we will check on each group's recent activity and make sure that everything is going ok.
+If a group is obviously inactive and does not respond to requests for a status update,
+or if desired by the organisers, the interest group will be archived.
+
+Anyone will be able to revive an archived special interest group at a future date,
+in a process much like proposing a new group.
+
+## Who can join?
+
+Anyone can join any interest group.
+Just as every nf-core pipeline is owned by the community, special interest groups
+will be made as inclusive as possible.
```

Comment:
I will join the two paragraphs as the one below is a consequence of the one above with something such as:
"""
Hence, we will try to avoid...
"""